management/post_firebase.js

```diff
@@ -0,0 +1,124 @@
+const requestPromise = require('request-promise');
+
+// [START get_sa_access_token]
+const {google} = require('googleapis');
+const SCOPES = ['https://www.googleapis.com/auth/firebase'];
+
+function getAccessToken() {
+  return new Promise(function(resolve, reject) {
```

Comment:
Use `(resolve, reject) =>` syntax rather than `function` syntax in Node.js snippets (apply throughout)

Reply:
Done